bootstrap/syncmanager.go

```diff
@@ -221,6 +221,7 @@ func (manager *SyncManager) syncBlockWorker(blockHeight int64) ([]command_entity
 	}
 
 	commands, err := parser.ParseBlockToCommands(
+		manager.logger,
```

Comment:
can pass new logger with block height field

Reply:
or I should use logger directly?